tsinfer/inference.py

```diff
@@ -469,7 +469,7 @@ def add_sites(self):
         logger.info("Starting addition of {} sites".format(self.num_sites))
         progress = self.progress_monitor.get("ga_add_sites", self.num_sites)
         for j, variant in enumerate(self.sample_data.variants(inference_sites=True)):
-            self.ancestor_builder.add_site(j, variant.site.time, variant.genotypes)
+            self.ancestor_builder.add_site(j, variant.inference_time, variant.genotypes)
```

Comment:
Not sure why we need the variant.inference_time field here?

Reply:
Variant.site.time will be `-inf` if no time was specified by the user. It is only the variant.inference_time that contains the frequency value of that genotype during iteration.